README.md

```diff
@@ -146,6 +146,15 @@ A separate Linebender integration for playing Lottie animations is available thr
 
 A separate Linebender integration for rendering raw scenes or Lottie and SVG files in [Bevy] through [`bevy_vello`](https://github.com/linebender/bevy_vello).
 
+### Cosmic Text
+
+An example scene demonstrating the integration of COSMIC text for font loading and text layout through [COSMIC Text Scene](https://github.com/linebender/vello/blob/main/examples/scenes/src/cosmic_text_scene.rs).
```

Comment:
This should probably use a relative path (so that e.g. people viewing this in their editor get the right path). It also lets us double check the link before merging.
See e.g. the links to the license files at the bottom of this file.

I do see that there are other cases in this file where that isn't done - it's fine not to fix that.